### PR TITLE
Fix: compose box height is changed after few seconds of app launch.

### PR DESCRIPTION
**Pull request by vishwesh3**

### CODE REVIEW

File: src/compose/ComposeBox.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export default class ComposeBox extends PureComponent<Props, State> {
     isMessageFocused: false,
     isTopicFocused: false,
     isMenuExpanded: false,
-    height: 40,
+    height: 23,
```
Comment (borisyankov) [inline src/compose/ComposeBox.js]: How did you come up with the value of 23? It is likely the correct value to be 30?
Considering this line:
```
const messageHeight = Math.min(Math.max(MIN_HEIGHT, height + 12), MAX_HEIGHT);
```

Comment (vishwesh3) [reply src/compose/ComposeBox.js]: Just after render [handleHeightChange](https://github.com/zulip/zulip-mobile/blob/e052865de90103d2922973113e1eb51d2bf9bf6e/src/compose/ComposeBox.js#L119) is called with height as `23`. (From here magic number 23 had came :)) Thus changing height from 42 with 23 and change in height of compose box. 

So now after `handleHeightChange` is called, height is same as before, thus we have also saved one re-render :)

Tested on iPhone 6, 5S, 8 Plus, X and it is working fine 😃

Comment (borisyankov) [reply src/compose/ComposeBox.js]: Fair enough :)

**Comment by timabbott** [comment]
For this sort of magic number thing, I'm a fan of putting a comment in both places noting they're linked (if it's not possible to just create a global constant both places use).

**Pull request merged**